### DISCUSSION TITLE
Problemi UPnP e SkyQ

### DISCUSSION THREAD

**Discussion by Stef62** [indomus/forum#915] [HUB: Home Assistant]
Buon pomeriggio a tutti.
Ho HA installato su docker raspberry versione 2021.10.0.

Dall'ultimo aggiornamento ho una marea di errori come questo (sono migliaia al giorno): 

`Logger: homeassistant
Source: /usr/src/homeassistant/homeassistant/runner.py:90
First occurred: 09:36:51 (1385 occurrences)
Last logged: 15:22:52
Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/async_upnp_client/search.py", line 83, in _async_on_data
    await self.async_callback(headers)
  File "/usr/local/lib/python3.9/site-packages/async_upnp_client/ssdp_listener.py", line 392, in _on_search
    ) = self._device_tracker.see_search(headers)
  File "/usr/local/lib/python3.9/site-packages/async_upnp_client/ssdp_listener.py", line 156, in see_search
    udn = headers["_udn"]
  File "/usr/local/lib/python3.9/site-packages/async_upnp_client/utils.py", line 57, in __getitem__
    return self._data[self._case_map[key.lower()]]
KeyError: '_udn'
`

Non avendo alcun dispositivo UPnP configurato in HA (se provo ad installare l'integrazione UPnP/IGD mi dice nessun dispositivo trovato), ho fatto il debug di "_async_upnp_client_" nel log e ho scoperto che l'errore viene generato dall'errato parsing XML interrogando lo SkyQ.
Nella mia rete c'è un NAS configurato DLNA che, leggendo il log, viene tranquillamente interrogato e riconosciuto mentre lo SkyQ mi da tutti quegli errori. 
Ora, visto che al momento non sono interessato a configurare dispositivi UPnP e non posso avere migliaia di errori nel log, è possibile trovare una soluzione tipo disabilitare il polling UPnP o al limite solo verso l'indirizzo dello SkyQ?
Grazie mille
Stefano

**Comment by snakuzzo** [comment]
Su quella versione c'era questo problema legato all'aggiornamento del modulo Python `async_upnp`. Il problema è stato risolto con la 2021.10.2 come puoi leggere [QUI](https://github.com/home-assistant/core/releases/tag/2021.10.2)
Quindi aggiorna Core e vedrai che gli errori scompariranno.

P.S. Prima di andare a caso, controllate le issue e, se possibile, restate sempre al passo con gli aggiornamenti

**Reply by Stef62** [reply]
Grazie mille.
Avevo fatto una ricerca sicuramente superficiale e non avevo notato che fosse uscita una nuova versione.
Provvedo subito a effettuare l'aggiornamento.
Grazie
Stefano
